addons/web_unsplash/static/src/components/media_dialog/image_selector.js

```diff
@@ -120,7 +120,7 @@ patch(ImageSelector.prototype, 'image_selector_unsplash', {
             return { records: [], isMaxed: false };
         }
         try {
-            const { isMaxed, images } = await this.unsplash.getImages(this.state.needle, offset, this.NUMBER_OF_ATTACHMENTS_TO_DISPLAY);
+            const { isMaxed, images } = await this.unsplash.getImages(this.state.needle, offset, this.NUMBER_OF_ATTACHMENTS_TO_DISPLAY, this.props.orientation);
```

Comment:
Does that orientation prop have a default of some kind? I don't see it here.

To avoid having cache data containing "aqueryexamplefalse" keys.

Reply:
Default is `undefined`, so I added a default in the `getImages()` prototype.
It indeed cached data as "aqueryexampleundefined"

Reply:
Another option would be to support the value being undefined and only append it to the cache key / to the unsplash payload if defined.